.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+## Related Issue
+
+<!-- Link related issue -->
+
+## Description
+
+<!-- Describe what this PR changes. -->
+
+## Review Procedure
+
+<!-- Explain how to review this PR. -->
+
+## Related PR(optional)
+
+<!-- Link related PR -->
+
+## Remarks(optional)
+
+<!-- Write remarks as you like if you need them. -->
+
+## Pre-Review Checklist for the PR Author
+
+**PR Author should check the checkboxes below when creating the PR.**
+
+- [ ] Check documentation
+- [ ] Check sample files
+- [ ] Read [commit-guidelines][commit-guidelines]
+- [ ] Assign PR to reviewer
+
+## Checklist for the PR Reviewer
+
+**Reviewers should check the checkboxes below before approval.**
+
+- [ ] Commits are properly organized and messages are according to the guideline
+- [ ] (Optional) Unit tests have been written for new behavior
```

Comment:
We still haven't defined the requirements for unit tests for Universe packages so I don't think we need this line at the moment. We can add them once we fix the requirements

Reply:
@mitsudome-r  remove at  46b5813